plugins/processors/t128_transform/t128_transform.go

```diff
@@ -250,6 +327,36 @@ func (r *T128Transform) Init() error {
 	return nil
 }
 
+func loadCache(path string) (map[uint64]map[string]observedValue, error) {
+	cache := make(map[uint64]map[string]observedValue)
+
+	data, err := ioutil.ReadFile(path)
+    if err != nil {
+      return cache, err
+    }
+
+	err = json.Unmarshal(data, &cache)
+    if err != nil {
+        return cache, err
+    }
+
+	return cache, nil
+}
+
+func persistCache(path string, cache map[uint64]map[string]observedValue) error {
+	file, err := json.MarshalIndent(cache, "", "  ")
```

Comment:
Prob don't need `MarshalIndent`

Reply:
i added the marshalIndent to make debugging/testing easier. That way it's nicer to visually inspect the file if needed

Reply:
Not saying you need to change this, but you may be solving a problem that can be solved other ways. For reference, I've found this very useful for such cases: `cat <the-file> | python -m json.tool`. That will produce nicely formatted JSON from the file.